### PR TITLE
chore: Add Erez as reviewer for .github

### PR DESCRIPTION
**Pull request by yevgenypats**
#### Summary

Add Erez as main reviewer for anything under .github. or maybe also add @yevgenypats @hermanschaaf ? (let me know what you think is best)?

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.github/*.  @erezrokah @hermanschaaf @yevgenypats
 cli/*       @yevgenypats
-plugins/source/* @cloudquery/cloudquery-opensource
\ No newline at end of file
+plugins/source/* @cloudquery/cloudquery-opensource
```
Comment (hermanschaaf) [inline CODEOWNERS:3]:
```suggestion
/plugins/source/ @cloudquery/cloudquery-opensource
```

To quote the [CODEOWNERS example from the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):

```
# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat
```

File: CODEOWNERS [hunk 2]
```diff
@@ -1,2 +1,3 @@
+.github/*.  @erezrokah @hermanschaaf @yevgenypats
```
Comment (hermanschaaf) [inline CODEOWNERS:1]:
```suggestion
/.github/  @erezrokah @hermanschaaf @yevgenypats
```

File: CODEOWNERS [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 cli/*       @yevgenypats
```
Comment (hermanschaaf) [inline CODEOWNERS:2]:
```suggestion
/cli/       @yevgenypats
```

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
I'm happy with being codeowner, but we should fix the syntax on all the lines 😄

**Review by erezrokah — APPROVED** [review]

**Pull request merged**